index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <main id="main">
+        <div class="header">
```

Comment:
I think it would be better to use semantic tags, because  Semantic tags provide meaningful structure to the content of a web page, making it easier for assistive technologies like screen readers to understand and interpret the content accurately.

Thank you, Happy coding!